### PR TITLE
Pin fixes & core-refactoring

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: src/core/dom/view.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+			layer.destroy();
+			layers.delete(name);
+			return true;
+		}
+
+		return false;
+	}
+
+	addClass(name){
+		if(this.markup.view){
```
Comment (etokareva) [inline src/core/dom/view.js]: Can you beautify code before push?

File: src/core/scroll/scroll.view.js [hunk 2]
```diff
@@ -13,20 +13,18 @@ export class ScrollView extends View {
...
-				f();
-				const container = this.y.container;
-				this.model.pagination({
-					current: Math.floor(container.position / model.pagination().size),
-					count: container.total
-				}, {
-					source: 'scroll.view',
-					behavior: 'core'
-				});
+		this.y.container.drawEvent.on(e => {
```
Comment (etokareva) [inline src/core/scroll/scroll.view.js:16]: Where we set drawEvent?

File: demo/pages/column-pin/index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 	<q-grid:columns generation="deep">
 		<q-grid:column key="name.last" pin="left"></q-grid:column>
 		<q-grid:column key="name.first" pin="left"></q-grid:column>
-		<q-grid:column key="options" pin="right" width="40" type="options" title="">
+		<q-grid:column key="options" pin="right" width="60" type="options" title="" can-resize="false">
```
Comment (etokareva) [inline demo/pages/column-pin/index.html:5]: "" can-resize="false" 
looks like typo in quotes

File: src/core/dom/cell.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import {Element} from './element';
+
+export class Cell extends Element {
+	constructor(context, element = null) {
+		super(element);
+
+		this.context = context;
+		//console.log(element);
```
Comment (etokareva) [inline src/core/dom/cell.js]: can be removed

File: src/core/scroll/scroll.view.js (hunk 2 above)
Comment (klumba12) [reply src/core/scroll/scroll.view.js:16]: Its in vscroll

File: demo/pages/column-pin/index.html (hunk 3 above)
Comment (klumba12) [reply demo/pages/column-pin/index.html:5]: m? ```title="" can-resize="false"```

Comment (etokareva) [reply demo/pages/column-pin/index.html:5]: Aaa, I see now ☺️

**Review by etokareva — APPROVED** [review]

**Pull request merged**